domains/bubitor.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "bubitor",
+        "email": "bubitor@proton.me"
+    },
+    "record": {
+        "A": ["130.61.232.64"]
```

Comment:
you should complete your website first, just "it works" isn't enough

Reply:
ah ok.. nevermind, thx